src/lib/svg_text_utils.js

```diff
@@ -106,9 +65,7 @@ exports.convertToTspans = function(_context, gd, _callback) {
                 'white-space': 'pre'
             });
 
-        result = _context.appendSVG(converted);
-
-        if(!result) _context.text(str);
+        _context.node().innerHTML = convertToSVG(str);
```

Comment:
http://caniuse.com/#search=innerHTML

![image](https://user-images.githubusercontent.com/6675409/27038362-c2258b88-4f58-11e7-9f84-dddea9dfe6c1.png)

Inconclusive. We'll have to try it on Browserstack.